tests/test_schema_generation.py

```diff
@@ -53,3 +54,14 @@ class TestStandardPythonSchemaGeneration(object):
     def test_standard_handling(self):
         assert get_input_schema(standard_py_func) == self.standard_sample_input_schema
         assert get_output_schema(standard_py_func) == self.standard_sample_output_schema
+
+
+class TestNestedSchemaGeneration(TestCase):
+    nested_sample_input_schema = json.loads(
+        resource_string(__name__, os.path.join('resources', 'sample_nested_input_schema.json')).decode('ascii'))
+    nested_sample_output_schema = json.loads(
+        resource_string(__name__, os.path.join('resources', 'sample_nested_output_schema.json')).decode('ascii'))
+
+    def test_nested_handling(self):
```

Comment:
I'm surprised this is passing given the missing input0/output0 handling in some places, makes me feel like something is off somewhere, or I'm misunderstanding something.

Reply:
Actually, it looks like it maybe isn't, but GitHub is saying all checks have passed. Will have to look into that